gtk/src/toga_gtk/libs/gtk.py

```diff
@@ -64,3 +64,15 @@
     from gi.repository import XApp  # noqa: F401
 except (ImportError, ValueError):  # pragma: no cover
     XApp = None
+
+try:
+    gi.require_version("Geoclue", "2.0")
+    from gi.repository import Geoclue  # noqa: F401
+except (ImportError, ValueError):  # pragma: no cover
+    Geoclue = None
+
+try:
+    gi.require_version("Flatpak", "1.0")
+    from gi.repository import Flatpak  # noqa: F401
+except (ImportError, ValueError):  # pragma: no cover
+    Flatpak = None
```

Comment:
I have not documented a dependency for this because in practice, it shouldn't be necessary. The Flatpak runtimes will already have the libraries required for this available (as they do for Geoclue), and the particular usage of this module in this PR is only relevant for an app running under Flatpak. _Maybe_ it would be relevant for a Snap build target, but Briefcase does not support Snap and I have no experience with Snap to know how it packages dependencies, or whether Snap's sandboxing even exposes the same errors when accessing location as Flatpak would.

TL;DR: No dependency needed for this, it'll be available when relevant under currently supported conditions.